src/generators.ts

```diff
@@ -10,4 +12,17 @@ export const generateSingleQueryMashup = (queryName: string, query: string): str
     shared #"${queryName}" = 
     ${query};`;
 
+export const generateMultipleQueryMashup = (loadedQuery: QueryInfo, queries: ConnectionOnlyQueryInfo[]): string => {
+    let mashup: string = generateSingleQueryMashup(loadedQuery.queryName!, loadedQuery.queryMashup!);
```

Comment:
What if you don't have a queryName? do you want to give a default name maybe? same as the mashup..

Reply:
Maybe you want to validate that queryName is not empty and if it is you can throw an exception